addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +42,60 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * Overrides values in addon/mixins/axis-titles.js
+  * @override
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
```

Comment:
Why does `xAxisPositionX` need to be recomputed when `labelWidthOffset` changes?